components/event-publisher-proxy/config/300-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: cloud-event-gateway-proxy
+  name: cloud-event-gateway-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloud-event-gateway-proxy
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: cloud-event-gateway-proxy
+    spec:
+      containers:
+        - env:
+            - name: CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: client-id
+            - name: CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: client-secret
+            - name: TOKEN_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: token-endpoint
+            - name: EMS_PUBLISH_URL
```

Comment:
I guess it confuses some people that we talk about OEB in the documentation, and use the acronym EMS here

Reply:
Sure, for now, it is EMS. Let's change it later when we decide on something else.